File: 프로그래머스/0/181833. 특별한 이차원 배열 1/특별한 이차원 배열 1.cs
using System;

public class Solution 
{
    public int[,] solution(int n) 
    {
        int[,] nArr = new int[n,n];
        
        for(int i=0; i<n; i++)
        {
            nArr[i,i] = 1;
        }
        return nArr;
    }
}

File: 프로그래머스/0/181833. 특별한 이차원 배열 1/특별한 이차원 배열 1.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(int n)
{
    vector<vector<int>> answer(n, vector<int>(n, 0));
    for(int i=0; i<answer.size(); i++)
    {
        answer[i][i] = 1;
    }
    return answer;
}